website/source/docs/platform/k8s/helm.html.md

```diff
@@ -377,6 +377,176 @@ to run the sync program.
+    reload the servers.
+
+  * <a name="v-meshgateway-globalmode" href="#v-meshgateway-globalmode">`globalMode`</a> (`string: "local"`) -
+  [Globally configure](/docs/connect/mesh_gateway.html#enabling-gateways-globally) which mode the gateway should run in.
+  Can be set to either "remote", "local", "none" or "".
+  See [Modes of Operation](/docs/connect/mesh_gateway.html#modes-of-operation) for a description of each mode.
+  If set, `connectInject.centralConfig.enabled` should be set to true so that
+  the global config will actually be used.
+  If set to the empty string, no global default will be set and the gateway mode
+  will need to be set individually for each service.
+
+  * <a name="v-meshgateway-replicas" href="#v-meshgateway-replicas">`replicas`</a> (`int: 2`) -
+  How many gateway pods to run.
+
+  * <a name="v-meshgateway-wanaddress" href="#v-meshgateway-wanaddress">`wanAddress`</a> -
+  The nested values define the WAN address registration that the `consul connect envoy` command will use to configure the mesh gateway.
+  See the [command overview](/docs/commands/connect/envoy.html#wan-address) for more details.
+
+      * <a name="v-meshgateway-wanaddress-port" href="#v-meshgateway-wanaddress-port">`port`</a> (`int: 443`) -
+      The port that gets registered for the WAN address.
+
+      * <a name="v-meshgateway-wanaddress-usenodeip" href="#v-meshgateway-wanaddress-usenodeip">`useNodeIP`</a> (`bool: true`) -
+      If true, each Gateway Pod will advertise its Node IP
+      (as provided by the Kubernetes downward API) as the WAN address.
+      This is useful if node IPs are routable from other DCs.
+      `useNodeName` and `host` must be `false` and `""` respectively.
+
+      * <a name="v-meshgateway-wanaddress-usenodename" href="#v-meshgateway-wanaddress-usenodename">`useNodeName`</a> (`bool: false`) -
+      If true, each Gateway Pod will advertise its Node Name
+      (as provided by the Kubernetes downward API) as the wan address.
+      This is useful if the node names are DNS entries that are routable from other DCs.
+      `useNodeIP` and `host` must be `false` and `""` respectively.
+
+      * <a name="v-meshgateway-wanaddress-host" href="#v-meshgateway-wanaddress-host">`host`</a> (`string: """`) -
+      If non-empty, each gateway Pod will use this host as its wan address.
+      Users must ensure that this address routes to the Gateway pods,
+      for example via a load balancer IP or a DNS entry that routes to the Service
+      fronting the Deployment.
+      `meshGateway.wanAddress.port` will be used as the port for the wan address.
+      `useNodeIP` and `useNodeName` must be false.
+
+  * <a name="v-meshgateway-service" href="#v-meshgateway-service">`service`</a> -
+  Values that configure the Kubernetes `Service` for the gateway `Deployment.
+
+      * <a name="v-meshgateway-service-enabled" href="#v-meshgateway-service-enabled">`enabled`</a> (`bool: false`) -
+      Whether to create a Service or not.
```

Comment:
This might be my own ignorance about K8s, but if creating a service for the gateways is optional it might be good to explain why you would or wouldn't do it.

Reply:
👍 will do